vendor/github.com/fgtdev/fortios-sdk-go/sdkcore/system_setting_global.go

```diff
@@ -16,6 +16,7 @@ type JSONSystemSettingGlobal struct {
 	Hostname     string `json:"hostname"`
 	AdminSport   string `json:"admin-sport"`
 	AdminSSHPort string `json:"admin-ssh-port"`
+	AdminScp     string `json:"admin-scp"`
```

Comment:
You really should be raising a PR against https://github.com/fgtdev/fortios-sdk-go, get that merged and tagged, _then_ vendor the changes to here for this PR.

Reply:
@davewongillies  Apologies, have created the same https://github.com/fgtdev/fortios-sdk-go/pull/2